### PR TITLE
Test that SSOT organisation test files are valid IATI XML

### PR DESCRIPTION
**Pull request by hayfield**
This shows that a couple were not. As such, they have been removed from pyIATI and the SSOT.

### CODE REVIEW

**Comment by hayfield** [comment]
Not very DRY :(

**Comment by hayfield** [comment]
The DRYness can't be resolved directly because #115 - refactoring is needed.

**Comment by allthatilk** [comment]
It's still tidy enough and readable enough that refactoring can wait until the validator API design is refined =)

File: iati/tests/test_validator.py [hunk 1]
```diff
@@ -415,12 +410,37 @@ def test_basic_validation_valid(self, schema_basic):
...
+
+        assert iati.validator.is_xml(dataset.xml_str)
+        assert iati.validator.is_iati_xml(dataset, schema)
+
+    @pytest.mark.parametrize('file_path', iati.tests.resources.get_test_data_paths_in_folder('ssot-activity-xml-fail') + iati.tests.resources.get_test_data_paths_in_folder('ssot-org-xml-fail'))
+    def test_basic_validation_should_fail_from_ssot(self, file_path, schema_basic, schema_basic_org):
+        """Perform check to see whether a parameter is valid IATI XML.
+
+        The parameter is valid IATI XML. It is sourced from the SSOT.
+        """
```
Comment (allthatilk) [inline iati/tests/test_validator.py]: This shouldn't have passed the linters. Did you lint this locally?

File: iati/tests/test_validator.py [hunk 2]
```diff
@@ -415,12 +410,37 @@ def test_basic_validation_valid(self, schema_basic):
         assert iati.validator.is_iati_xml(data, schema_basic)
         assert iati.validator.is_valid(data, schema_basic)
 
-    def test_basic_validation_should_pass_from_ssot(self, iati_dataset_valid_from_ssot, schema_basic):
+    @pytest.mark.parametrize('file_path', iati.tests.resources.get_test_data_paths_in_folder('ssot-activity-xml-pass') + iati.tests.resources.get_test_data_paths_in_folder('ssot-org-xml-pass'))
+    def test_basic_validation_should_pass_from_ssot(self, file_path, schema_basic, schema_basic_org):
         """Perform check to see whether a parameter is valid IATI XML.
 
         The parameter is valid IATI XML. It is sourced from the SSOT.
         """
```
Comment (allthatilk) [inline iati/tests/test_validator.py]: Also linting.

File: iati/tests/test_validator.py (hunk 1 above)
Comment (hayfield) [reply iati/tests/test_validator.py]: linting is all passing

Comment (hayfield) [reply iati/tests/test_validator.py]: http://www.pydocstyle.org/en/2.1.1/error_codes.html indicates `D410`requires a blank line after a section. This docstring has no sections, so all is well!

**Pull request merged**